tests/snapshots/assets__check_file@packages-tidy-new-parser.typ-80.snap

```diff
@@ -257,10 +257,9 @@ snapshot_kind: text
     if args.len() > 0 and (
       state.unfinished-param.ends-with(",") or state.state == "finished"
     ) {
-      state.params.push((
-        name: args.first(),
-        desc-lines: state.unmatched-description,
-      ))
+      state
+        .params
+        .push((name: args.first(), desc-lines: state.unmatched-description))
       state.unmatched-description = ()
```

Comment:
The previous one looks better. But I think this is out of this PR's scope. We need determine whether to enter chain mode depending on the length of the chain itself(without parameters of the func call)